### ISSUE TITLE
ADA: No data cells assigned to table header

### ISSUE THREAD

**Issue by fpichardom** [AustinMastLab/ckanext-digitizationknowledge#43]
**Description:** Every header cell in a table should have at least one data cell assigned to it.
**Related to issue ADA Title II, EAA**

- WCAG 2.2: https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships
- WCAG 2.2: https://www.w3.org/WAI/WCAG22/Techniques/html/H43
- Siteimprove Alfa: https://alfa.siteimprove.com/rules/sia-r46

**Comment by rbruhn** [comment]
Tests complete and showing 100%
